graphql-client/src/main/java/com/expediagroup/streamplatform/streamregistry/graphql/client/StreamRegistryClient.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (C) 2018-2019 Expedia, Inc.
+ * Copyright (C) 2018-2020 Expedia, Inc.
```

Comment:
The IDE is changing files automagically when I open some source files. I thought update the year wasn't problem, that's why I let it. I'll configure it to avoid that behavior.